lemur/certificates/schemas.py

```diff
@@ -112,10 +112,20 @@ def load_data(self, data):
         if data.get('replacements'):
             data['replaces'] = data['replacements']  # TODO remove when field is deprecated
         if data.get('csr'):
-            dns_names = cert_utils.get_dns_names_from_csr(data['csr'])
-            if not data['extensions']['subAltNames']['names']:
+            csr_sans = cert_utils.get_sans_from_csr(data['csr'])
+            if not data.get('extensions'):
+                data['extensions'] = {
+                    'subAltNames': {
+                        'names': []
+                    }
+                }
+            elif not data['extensions'].get('subAltNames'):
+                data['extensions']['subAltNames'] = {
+                    'names': []
+                }
+            elif not data['extensions']['subAltNames']['names']:
```

Comment:
`elif not data['extensions']['subAltNames'].get('names'):`

Reply:
I've added 58dd424de8781e852fbb4fac102273746120e5e2 to prevent this case, thanks!